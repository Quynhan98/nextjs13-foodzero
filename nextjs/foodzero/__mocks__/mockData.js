export const AVATAR_MOCK =
  'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2Fjohn-doe.webp?alt=media&token=6963f744-1ab8-4b09-971d-2486cd5375bf'

export const PRICE_LIST_MOCK = {
  name: 'Deep Sea Snow White Cod Fillet',
  price: 20,
  description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
}

export const CARD_BLOG = {
  imageUrl:
    'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2FFeature%20Image.webp?alt=media&token=0a725961-6c86-4dcd-ab48-e5d7e2eb48fd',
  title: 'Fruit and vegetables and protection against diseases',
  description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
  category: 'Fashion',
  srcAvatar: AVATAR_MOCK,
  name: 'Julie Christie',
  date: '2022-12-06T23:29:35.000Z',
  comments: 2,
}

export const CARD_FEATURE = {
  title: 'Premium Quality',
  imageUrl:
    'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2FCircleIcon.webp?alt=media&token=1df373f3-27a3-45f6-82fa-d997c47c718f',
  description:
    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Neque congue arcu',
}

export const CARD_CATEGORY = {
  href: '/',
  category: 'Starters',
  imageUrl:
    'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2Fcategory.webp?alt=media&token=4e7dcf6c-2a66-4695-917d-af1d578d9a30',
}

export const USERS_MOCK = [
  {
    id: 'gff1',
    email: 'foodzero@gmail.com',
    password: '123456789',
    reservations: [],
  },
]

export const QUOTE_MOCK = [
  {
    id: '1',
    authorName: 'John Doe',
    job: 'Blogger',
    quote:
      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Purus lorem id penatibus impedient. Turpis egestas ultricies purus  Lorem ipsum dolor sit amet.',
    avatar:
      'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2Fjohn-doe.webp?alt=media&token=6963f744-1ab8-4b09-971d-2486cd5375bf',
  },
  {
    id: '2',
    authorName: 'Julie Christie',
    job: 'Blogger',
    quote:
      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Purus lorem id penatibus impedient. Turpis egestas ultricies purus  Lorem ipsum dolor sit amet1.',
    avatar:
      'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2Funnamed.webp?alt=media&token=f2607e25-62ae-4ed9-9f53-bb76e6f830fb',
  },
  {
    id: '3',
    authorName: 'Dianne Russell',
    job: 'Blogger',
    quote:
      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Purus lorem id penatibus impedient. Turpis egestas ultricies purus  Lorem ipsum dolor sit amet2.',
    avatar:
      'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2Fcute-avatar.webp?alt=media&token=9d70884c-e075-4f36-bf67-b196493badb8',
  },
]

export const OUR_MENU_MOCK = [
  {
    id: '1',
    name: 'Deep Sea Snow White Cod Fillet',
    price: 20,
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit1.',
  },
  {
    id: '2',
    name: 'Steak With Rosemary Butter',
    price: 22,
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit2.',
  },
  {
    id: '3',
    name: 'Cucumber Salad',
    price: 18,
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit3.',
  },
  {
    id: '4',
    name: 'Natural Wine Pairing',
    price: 90,
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit4.',
  },
]

export const BLOG_SECTION_MOCK = [
  {
    imageUrl:
      'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2FFeature%20Image.webp?alt=media&token=766a34ea-32e0-40e7-90e6-299f9af22eb1',
    title: 'Fruit and vegetables and protection against diseases',
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit1.',
    category: 'Fashion',
    srcAvatar:
      'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2Fjulie.webp?alt=media&token=c429d5f2-6f41-4233-970b-b2739746f7f1',
    name: 'Julie Christies',
    date: '2021-10-17T23:29:35.000Z',
    comments: 2,
    id: '1',
  },
  {
    imageUrl:
      'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2Fasparagus.webp?alt=media&token=d56ab468-1479-4181-844d-f5b231742244',
    title: "Asparagus Spring Salad with Rocket, Goat's Cheese",
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit2.',
    category: 'Fashion',
    srcAvatar:
      'https://firebasestorage.googleapis.com/v0/b/image-blog-155e2.appspot.com/o/foodzero%2Fdianne.webp?alt=media&token=ce3d7213-72ce-4caa-b43c-f6f07bb3b5b7',
    name: 'Dianne Russell',
    date: '2021-10-17T23:29:35.000Z',
    comments: 2,
    id: '2',
  },
]

export const MENU_MOCK = {
  id: '1',
  ourMenu: OUR_MENU_MOCK,
  starters: OUR_MENU_MOCK,
  mains: OUR_MENU_MOCK,
  pastriesAndDrinks: OUR_MENU_MOCK,
}
