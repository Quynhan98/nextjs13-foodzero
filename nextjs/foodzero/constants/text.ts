export const SNACKBAR_BOOKING_SUCCESS =
  'You have made a successful reservation.'
