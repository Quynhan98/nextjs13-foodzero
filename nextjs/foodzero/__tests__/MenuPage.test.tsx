// Utils
import {
  cleanup,
  screen,
  render,
  RenderResult,
  fireEvent,
  waitFor,
} from '@utils/testUtils'

// Mocks
import { MENU_MOCK } from '@mocks/mockData'

// Constants
import { SNACKBAR_BOOKING_SUCCESS } from '@constants/text'
import { SERVER_ERROR } from '@constants/index'

// App
import MenuPage from 'app/(landing)/menu/MenuPage'

describe('Menu page', () => {
  global.fetch = jest.fn(() =>
    Promise.resolve({
      json: () => Promise.resolve(),
    }),
  ) as jest.Mock

  const props = { menu: MENU_MOCK }

  const bookingMock = {
    date: '12/21/2022',
    time: '12:00 am',
    person: '2 Person',
  }

  let wrapper: RenderResult

  beforeEach(() => {
    wrapper = render(<MenuPage content={JSON.stringify(props)} />)
  })

  afterEach(() => {
    cleanup()
  })

  it('Renders Menu page correctly', () => {
    const { container } = wrapper

    expect(container).toMatchSnapshot()
  })

  it('Should call onChange date picker', () => {
    const date = screen.getByPlaceholderText('MM/DD/YYYY')

    fireEvent.change(date, {
      target: { value: bookingMock.date },
    })

    expect(date).toHaveValue(bookingMock.date)
  })

  it('Should onchange time ', () => {
    const time = screen.getByTestId('selectTime')

    fireEvent.change(time, { target: { value: bookingMock.time } })

    expect(time).toHaveValue(bookingMock.time)
  })

  it('Should onchange person ', () => {
    const person = screen.getByTestId('selectPerson')

    fireEvent.change(person, { target: { value: bookingMock.person } })

    expect(person).toHaveValue(bookingMock.person)
  })

  it('Should call submit form booking', () => {
    const button = screen.getByRole('button', { name: /Book Now/i })

    fireEvent.submit(button)

    waitFor(() => {
      expect(screen.getByText(SNACKBAR_BOOKING_SUCCESS)).toBeTruthy()
    })
  })

  it('Should call submit form booking with server error', () => {
    jest.fn().mockImplementationOnce(() => Promise.reject(SERVER_ERROR))

    const button = screen.getByRole('button', { name: /Book Now/i })

    fireEvent.submit(button)

    waitFor(() => {
      expect(screen.getByText(SERVER_ERROR)).toBeTruthy()
    })
  })
})
