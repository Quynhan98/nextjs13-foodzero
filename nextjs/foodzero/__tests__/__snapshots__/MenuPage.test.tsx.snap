// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu page Renders Menu page correctly 1`] = `
<div>
  <div
    class="css-7obpig"
  >
    <h2
      class="chakra-heading css-2ka32n"
    >
      View Our New Menu
    </h2>
    <p
      class="chakra-text css-rkgsxi"
    >
      The freshest ingredients for you every day
    </p>
  </div>
  <section
    class="css-mjcp4"
  >
    <h3
      class="chakra-heading css-14bagvu"
    >
      Starters
    </h3>
    <p
      class="chakra-text css-ot837d"
    >
      This is a section of your menu. Give your section a brief description
    </p>
    <div
      class="css-1qe38uk"
    >
      <div
        class="css-1wh39tp"
      >
        <img
          alt="excellent cook picture"
          data-nimg="fill"
          decoding="async"
          sizes="(max-width: 768px) 277px, 314px (min-width: 768px) 554px, 629px (min-width: 1200px) 792px, 898px"
          src="/_next/image?url=%2Fimages%2FyoungTofu.webp&w=3840&q=75"
          srcset="/_next/image?url=%2Fimages%2FyoungTofu.webp&w=16&q=75 16w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=32&q=75 32w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=48&q=75 48w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=64&q=75 64w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=96&q=75 96w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=128&q=75 128w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=256&q=75 256w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=384&q=75 384w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=640&q=75 640w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=750&q=75 750w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=828&q=75 828w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FyoungTofu.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
      <ul
        class="css-yjlmq8"
        role="list"
      >
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="1"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              20
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Deep Sea Snow White Cod Fillet
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit1.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="2"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              22
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Steak With Rosemary Butter
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit2.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="3"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              18
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Cucumber Salad
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit3.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="4"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              90
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Natural Wine Pairing
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit4.
            </p>
          </div>
        </li>
      </ul>
    </div>
  </section>
  <section
    class="css-oklxer"
  >
    <h3
      class="chakra-heading css-14bagvu"
    >
      Mains
    </h3>
    <p
      class="chakra-text css-ot837d"
    >
      This is a section of your menu. Give your section a brief description
    </p>
    <div
      class="css-ijaihs"
    >
      <ul
        class="css-1fvuai0"
        role="list"
      >
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="1"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              20
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Deep Sea Snow White Cod Fillet
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit1.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="2"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              22
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Steak With Rosemary Butter
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit2.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="3"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              18
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Cucumber Salad
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit3.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="4"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              90
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Natural Wine Pairing
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit4.
            </p>
          </div>
        </li>
      </ul>
      <div
        class="css-1wh39tp"
      >
        <img
          alt="excellent cook picture"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          sizes="(max-width: 768px) 277px, 314px (min-width: 768px) 554px, 629px (min-width: 1200px) 792px, 898px"
          src="/_next/image?url=%2Fimages%2Fsteak.webp&w=3840&q=75"
          srcset="/_next/image?url=%2Fimages%2Fsteak.webp&w=16&q=75 16w, /_next/image?url=%2Fimages%2Fsteak.webp&w=32&q=75 32w, /_next/image?url=%2Fimages%2Fsteak.webp&w=48&q=75 48w, /_next/image?url=%2Fimages%2Fsteak.webp&w=64&q=75 64w, /_next/image?url=%2Fimages%2Fsteak.webp&w=96&q=75 96w, /_next/image?url=%2Fimages%2Fsteak.webp&w=128&q=75 128w, /_next/image?url=%2Fimages%2Fsteak.webp&w=256&q=75 256w, /_next/image?url=%2Fimages%2Fsteak.webp&w=384&q=75 384w, /_next/image?url=%2Fimages%2Fsteak.webp&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fsteak.webp&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fsteak.webp&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fsteak.webp&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fsteak.webp&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fsteak.webp&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fsteak.webp&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fsteak.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
    </div>
  </section>
  <section
    class="css-17zsyi1"
  >
    <h3
      class="chakra-heading css-14bagvu"
    >
      Pastries & Drinks
    </h3>
    <p
      class="chakra-text css-ot837d"
    >
      This is a section of your menu. Give your section a brief description
    </p>
    <div
      class="css-1yhbdt5"
    >
      <div
        class="css-1wh39tp"
      >
        <img
          alt="excellent cook picture"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          sizes="(max-width: 768px) 277px, 314px (min-width: 768px) 554px, 629px (min-width: 1200px) 792px, 898px"
          src="/_next/image?url=%2Fimages%2FlemonTea.webp&w=3840&q=75"
          srcset="/_next/image?url=%2Fimages%2FlemonTea.webp&w=16&q=75 16w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=32&q=75 32w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=48&q=75 48w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=64&q=75 64w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=96&q=75 96w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=128&q=75 128w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=256&q=75 256w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=384&q=75 384w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=640&q=75 640w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=750&q=75 750w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=828&q=75 828w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FlemonTea.webp&w=3840&q=75 3840w"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
      <ul
        class="css-yjlmq8"
        role="list"
      >
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="1"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              20
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Deep Sea Snow White Cod Fillet
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit1.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="2"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              22
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Steak With Rosemary Butter
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit2.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="3"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              18
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Cucumber Salad
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit3.
            </p>
          </div>
        </li>
        <li
          class="css-0"
        >
          <div
            class="css-0"
            id="4"
          >
            <h4
              class="chakra-heading css-4jbmzn"
            >
              $
              90
            </h4>
            <hr
              class="css-113h5p7"
            />
            <h3
              class="chakra-heading css-u3uye4"
            >
              Natural Wine Pairing
            </h3>
            <p
              class="chakra-text css-1oh60g1"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit4.
            </p>
          </div>
        </li>
      </ul>
    </div>
  </section>
  <section
    class="css-9baf2d"
    id="reservationSection"
  >
    <form
      novalidate=""
    >
      <div
        class="chakra-form-control css-qjxwnb"
        role="group"
      >
        <div
          class="css-yjtq8f"
        >
          <h3
            class="chakra-heading css-17j5m7v"
          >
            Make a Reservation
          </h3>
          <p
            class="chakra-text css-1ce86fs"
          >
            Get in touch with restaurant
          </p>
        </div>
        <div
          class="css-4yycd6"
        >
          <div
            class="css-1payax8"
          >
            <div
              class="css-0"
            >
              <div
                class="react-datepicker-wrapper"
              >
                <div
                  class="react-datepicker__input-container"
                >
                  <input
                    class="chakra-input css-1sz8sj0"
                    id="field-:r0:"
                    placeholder="MM/DD/YYYY"
                    value=""
                  />
                </div>
              </div>
            </div>
            <div
              class="css-0"
            >
              <div
                class="chakra-select__wrapper css-17qe6bp"
              >
                <select
                  class="chakra-select css-lknr9b"
                  data-testid="selectTime"
                  id="field-:r0:"
                  name="time"
                  style="padding-left: 40px;"
                >
                  <option
                    value=""
                  >
                    11:00 am
                  </option>
                  <option
                    value="11:00 am"
                  >
                    11:00 am
                  </option>
                  <option
                    value="12:00 am"
                  >
                    12:00 am
                  </option>
                  <option
                    value="13:00 pm"
                  >
                    13:00 pm
                  </option>
                  <option
                    value="14:00 pm"
                  >
                    14:00 pm
                  </option>
                  <option
                    value="15:00 pm"
                  >
                    15:00 pm
                  </option>
                  <option
                    value="16:00 pm"
                  >
                    16:00 pm
                  </option>
                  <option
                    value="17:00 pm"
                  >
                    17:00 pm
                  </option>
                  <option
                    value="18:00 pm"
                  >
                    18:00 pm
                  </option>
                  <option
                    value="19:00 pm"
                  >
                    19:00 pm
                  </option>
                </select>
                <div
                  class="chakra-select__icon-wrapper css-1rn5cz7"
                >
                  <svg
                    aria-hidden="true"
                    class="chakra-icon chakra-select__icon css-m73ylg"
                    focusable="false"
                    role="presentation"
                    style="width: 1em; height: 1em; color: currentColor;"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
          <div
            class="chakra-select__wrapper css-42b2qy"
          >
            <select
              class="chakra-select css-lknr9b"
              data-testid="selectPerson"
              id="field-:r0:"
              name="person"
              style="padding-left: 40px;"
            >
              <option
                value=""
              >
                1 Person
              </option>
              <option
                value="1 Person"
              >
                1 Person
              </option>
              <option
                value="2 Person"
              >
                2 Person
              </option>
              <option
                value="3 Person"
              >
                3 Person
              </option>
              <option
                value="4 Person"
              >
                4 Person
              </option>
              <option
                value="5 Person"
              >
                5 Person
              </option>
              <option
                value="6 Person"
              >
                6 Person
              </option>
              <option
                value="7 Person"
              >
                7 Person
              </option>
              <option
                value="8 Person"
              >
                8 Person
              </option>
              <option
                value="9 Person"
              >
                9 Person
              </option>
              <option
                value="10 Person"
              >
                10 Person
              </option>
            </select>
            <div
              class="chakra-select__icon-wrapper css-1rn5cz7"
            >
              <svg
                aria-hidden="true"
                class="chakra-icon chakra-select__icon css-m73ylg"
                focusable="false"
                role="presentation"
                style="width: 1em; height: 1em; color: currentColor;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  fill="currentColor"
                />
              </svg>
            </div>
          </div>
        </div>
        <div
          class="css-1nx6xr5"
        >
          <button
            class="chakra-button css-1y4zi6m"
            disabled=""
            type="submit"
          >
            Book Now
          </button>
        </div>
      </div>
    </form>
  </section>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
